src/components/ui/experimentSpeechBubble.module.css

```
@@ -0,0 +1,41 @@
+.speechBubble {
+  width: 150px;
+  height: 100px;
+  background-color: white;
+  border: 2px solid darkred;
+  border-radius: 50%;
+  position: relative;
+  display: inline-block;
+}
+
+.arrow {
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-left: 15px solid transparent;
+  border-right: 15px solid transparent;
+  border-top: 15px solid white; /* 矢印を白に変更 */
```

文脈ありきかつ当時の AI のコメントな気がするのでいらなさそう
```suggestion
  border-top: 15px solid white;
```